modules/custom/openy_daxko2/src/Form/SettingsForm.php

```diff
@@ -70,6 +71,27 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => t('Refresh token is a large string like 241 chars long.'),
     ];
 
+    $form['locations'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Locations'),
+      '#default_value' => $config->get('locations'),
+      '#description' => t('Locations mapping. One per row. "<location id>,<location name>". To group Locations wrap name of the location with asterix: *Camps*.'),
+    ];
+
+    $form['categories'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Categories'),
+      '#default_value' => $config->get('categories'),
+      '#description' => t('Categories. One per row. "<category id>,<category name>". To group categories wrap name of the category with asterix: *Swimming*.'),
+    ];
+
+    $form['percentage'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Percentage'),
+      '#default_value' => !empty($config->get('percentage')) ? $config->get('percentage') : 100,
+      '#description' => t('Percent of similarity between Session and Program names for displaying (100 means only identical names will be combined).'),
```

Comment:
Any recommendation to when it it not 100% - some practical use case here in description